system-test/fixtures/google-gax-packaging-test-app/protos/protos.json

```diff
@@ -0,0 +1,1281 @@
+{
+  "nested": {
```

Comment:
Also, if we add `protos.json`, we might want to remove the old `echo.json`.

Reply:
Done. Got rid of the client as well.